src/main/java/com/ust/sourcecourse/configuration/exception/CustomException.java

```diff
@@ -0,0 +1,48 @@
+package com.ust.sourcecourse.configuration.exception;
+
+import java.util.List;
+
+public class CustomException extends RuntimeException {
```

Comment:
Have each exception classes separately instead of inner classes. And remove CustomException class if not required. If used, rename it to a proper name.

Reply:
resolved